### ISSUE TITLE
Check: CKV_GCP_69: "Ensure the GKE Metadata Server is Enabled"

### ISSUE THREAD

**Issue by brettcurtis** [bridgecrewio/checkov#4266]
**Describe the issue**
Is it reasonable to say I'm hitting a false positive here if I have the following enabled :

```hcl
  workload_metadata_config {
    mode = "GKE_METADATA_SERVER"
  }
```

https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/container_cluster.html#workload_metadata_config

**Version (please complete the following information):**
 - 2.2.255

**Comment by brettcurtis** [comment]
Bah user error!
